### PR TITLE
Fixes missing trailing line and delete space.

### PR DESCRIPTION
**Pull request by DeadLineSS13**

### CODE REVIEW

**Comment by DeadLineSS13** [comment]
Squash and merge this.

File: baystation12.dme [hunk 1]
```diff
@@ -1,3097 +1,3097 @@
...
-#include "maps\~mapsystem\map_ranks.dm"
-#include "maps\~mapsystem\maps.dm"
-#include "maps\~mapsystem\maps_announcements.dm"
-#include "maps\~mapsystem\maps_areas.dm"
-#include "maps\~mapsystem\maps_jobs.dm"
-#include "maps\~mapsystem\maps_unit_testing.dm"
-#include "maps\~unit_tests\unit_testing.dm"
-#include "~code\global_init.dm"
-// END_INCLUDE
+// DM Environment file for baystation12.dme.
```
Comment (NanakoAC) [inline baystation12.dme]: why are there any changes to this file at all?

File: code/__defines/_planes+layers.dm [hunk 2]
```diff
@@ -237,7 +237,7 @@ What is the naming convention for planes or layers?
 	layer = initial(layer)
 
 /*
-  PLANE MASTERS
```
Comment (NanakoAC) [inline code/__defines/_planes+layers.dm:240]: this kind of thing is done deliberately for formatting. Whitespace inside comments should not be touched

File: code/controllers/failsafe.dm [hunk 3]
```diff
@@ -1,8 +1,8 @@
- /**
-  * Failsafe
```
Comment (NanakoAC) [inline code/controllers/failsafe.dm:2]: messing with things in comments again, theres a lot more of this

File: code/controllers/subsystems/initialization/codex.dm [hunk 4]
```diff
@@ -113,9 +113,9 @@ SUBSYSTEM_DEF(codex)
 			for(var/entry_title in entries_by_string)
 				var/datum/codex_entry/entry = entries_by_string[entry_title]
 				if(findtext(entry.display_name, searching) || \
-				 findtext(entry.lore_text, searching) || \
-				 findtext(entry.mechanics_text, searching) || \
-				 findtext(entry.antag_text, searching))
+					findtext(entry.lore_text, searching) || \
```
Comment (NanakoAC) [inline code/controllers/subsystems/initialization/codex.dm:116]: this doesnt seem correct, these are all part of the same IF and shouldn't be indented. They're not inside the conditional area

**Review by NanakoAC — CHANGES_REQUESTED** [review]
I'm only a little bit in and i've found too many problems to continue.

File: code/__defines/_planes+layers.dm (hunk 2 above)
Comment (DeadLineSS13) [reply code/__defines/_planes+layers.dm:240]: [space] can give error then [tab] in run_linters

**Comment by NanakoAC** [comment]
This PR will not be merged. As we discussed a few days ago, it is too big to review, or to merge without reviewing.

Please make a new and much smaller PR that includes only the manual work. The things done by a mass editing script, i can do personally. we need this broken into parts

**Pull request closed without merging**